spec/Vehicle/Vehicle.vspec

```diff
@@ -175,12 +175,10 @@
   description: Accumulated idle time in seconds.
 
 - Speed:
-  datatype: int32
+  datatype: float
   type: sensor
-  min: -250
-  max: 250
   unit: km/h
-  description: Vehicle speed, as sensed by the gearbox.
+  description: Vehicle speed
```

Comment:
We recently marked "Navigation.CurrentLocation.Speed" as deprecated with the rationale that it was a duplicate. But for that to be true Vehicle.Speed must be agnostic to how speed is sensed/calculated, i.e. we shall remove "as sensed by the gearbox" here?

Reply:
Today we discussed GNSS-speed vs. speed as measured by the wheels, and in corner cases there could be big differences. For example, if your car is sliding down an icy road - then actual speed might be higher than wheel speed? If the car is tested on a "rolling road", GPS speed and wheel speed will be very different.

Do we see a benefit in adding detailed documentation on how each datapoint shall be interpreted, or do we prefer leaving that to each OEM? We could go "all in" and try to detail something like below, but I am not sure that we want that:

```
# The vehicle may use any method to provide this value (e.g. GPS, Radar, gearbox-sensor, wheel sensor)
# If possible, this sensor value shall represent the true longitudinal speed of the vehicle compared to the ground.
# It is acceptable that this is not always achievable for certain cases 
# Example 1: If the car is tested on a "rolling road", it is preferable that the datapoint reports 0 as speed, 
#                   but it is acceptable if a speed based on wheel or gearbox rotations are reported
# Example 2: If the car is parked on a moving ferry,  it is preferable that the datapoint reports 0 as speed, 
#                   but it is acceptable if a speed based on GPS is reported

```